usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -68,6 +68,7 @@ SELECT
   vw_award_search.pop_zip5,
   vw_award_search.pop_congressional_code,
   vw_award_search.pop_city_name,
+  vw_award_search.pop_city_code,
```

Comment:
What about the addition of `vw_award_search.recipient_location_city_code` ?

Reply:
I was basing the fields on the lookups in `matview_lookups.py`, which only included the `pop_city_code`. We can include the `recipient_location_city_code` as well, it's just not currently included.

Reply:
Gotcha. The lookups are for the table columns, but not the search/filter operations. It is possible `recipient_location_city_code` isn't necessary like `pop_city_code`.